qiskit/providers/aer/noise/noise_model.py

```diff
@@ -1045,3 +1063,34 @@ def _nonlocal_quantum_errors_equal(self, other):
                     if iinner_dict1[iinner_key] != iinner_dict2[iinner_key]:
                         return False
         return True
+
+    def pass_manager(self, only_custom: bool = False) -> Optional[PassManager]:
```

Comment:
We could just return a trival (empty) pass manager for the case where there are no passes. Then later you can always assume you can do `pm.run(circuits)`.

This would require changes to pass manager that if it has no passes it just returns the input circuits 
```python
class PassManager:
   ...
    def run(
        self,
        circuits: Union[QuantumCircuit, List[QuantumCircuit]],
        output_name: str = None,
        callback: Callable = None,
    ) -> Union[QuantumCircuit, List[QuantumCircuit]]:
    if not self._pass_set and output_name is None and callback is None:
        return circuits
    # current code...
```

otherwise it will have bad performance

Reply:
I would also suggest leaving this as in internal method for this release until we have implemented converting the full noise model to a pass manager (which needs to either handle non-local errors or wait till they have been removed after deprecation period as mentioned below)
```suggestion
    def _pass_manager(self) -> Optional[PassManager]:
```

Reply:
Yeah, as you pointed out, I wanted to return empty pass manager but I checked the implementation of the current PassManager and ended up with returning None in the case. In 3f6239a8, I've kept the return type optional but made the function private. So we can change not to return None without deprecation after improving the PassManager implementation.

Reply:
We can change it to an empty pass manger after release. I made a PR to Terra which will be included in the 0.19 release to make pass managers return the input circuits if there are no passes so performance shouldn't be an issue